packages/core/src/linker/view_container_ref.ts

```diff
@@ -210,7 +210,7 @@ export abstract class ViewContainerRef {
   ): EmbeddedViewRef<C>;
 
   /**
-   * Instantiates a single component and inserts its host view into this container.
+   * Instantiates a given component, inserts its host view into this view container and updates the DOM.
```

Comment:
I think this is a bit verbose and isn't that much clearer than what was there before.

```suggestion
   * Instantiates a component and inserts its host view into this view container.
```

Reply:
this was originally suggested by @AndrewKushnir at https://github.com/angular/angular/pull/60359#pullrequestreview-2711826186